src/Appwrite/Utopia/Fetch/BodyMultipart.php

```diff
@@ -136,7 +136,7 @@ public function exportBody(): string
             }
 
             $query .= $eol . $eol;
-            $query .= $value . $eol;
+            $query .= $value == false ? 0 . $eol : $value . $eol;
```

Comment:
Should we do `===` instead of `==`? Not sure if other falsy values like empty array or string should be `0`